pages/index.tsx

```diff
@@ -1,33 +1,117 @@
 import * as React from 'react';
-import type { NextPage } from 'next';
+import {GetServerSideProps, NextPage} from "next";
+import Head from "next/head";
+import {ExperienceWithContent, getExperience} from "./api/experience/[shortcode]";
 import Container from '@mui/material/Container';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
-import Link from 'src/components/Link';
+import {Button} from "@mui/material";
+import LoadingButton from '@mui/lab/LoadingButton'
+import Paper from "@mui/material/Paper";
+import Image from 'next/image';
+import {useRouter} from "next/router";
+import {useState} from "react";
+import superjson from "superjson";
+
+
+type ExperiencePageProps = {
+    experience: ExperienceWithContent
+}
+
+const Home: NextPage<ExperiencePageProps> = ({experience}) => {
+
+    const router = useRouter();
+    const [loading, setLoading] = useState(false);
+
+    const loadingCuriousExperience = () => {
+        //onClick for "Launch Quiz", loading indicator occurs while waiting to be pushed to '/curious' page
+       setLoading(true);
+       router.push('/curious');
+    }
 
-const Home: NextPage = () => {
   return (
-      <Container maxWidth="lg">
-        <Box
-            sx={{
-              my: 4,
-              display: 'flex',
-              flexDirection: 'column',
-              justifyContent: 'center',
-              alignItems: 'center',
-            }}
+
+      <Container maxWidth="md"
+                sx={{
+                   my: 4,
+                   display: 'flex',
+                   flexDirection: 'column',
+                   justifyContent: 'center',
+                   alignItems: 'center',
+                }}
         >
-          <Typography variant="h4" component="h1" gutterBottom>
-            MUI v5 + Next.js with TypeScript example
-          </Typography>
-          <Link href="/quiz" color="secondary">
-            Go to the about page
-          </Link>
-            <img src="/api/quiz/gauge.png?min=5&max=15&value=8" alt="the gauge"/>
-            <img src="/api/quiz/summary.png?percents=[10, 30, 50, 10]&total=245" alt="the results"/>
-        </Box>
+            <Head><title>Experience Engine</title></Head>
+            <Typography variant="h2" component="h1" fontWeight={500}>{experience.name}</Typography>
+            <Typography variant="h4" component="h2">{experience.subtitle}</Typography>
+
+          <Paper
+              elevation={3}
+              sx={{
+                 p:5,
+                 m:1
+              }}
+          >
+             <Image src={`/images/${experience.shortcode}/1.png`} width={500} height={333}  alt="Girl with Magnifying Glass"/>
+             <Box height="55px" position="relative"
+             >
+                {/*if loading is true, display loading indicator; is false, display 'Launch Quiz' button*/}
+                 {loading
```

Comment:
This looks pretty cool when it's loading. I wonder if the loading code is common enough that you could make a component for it? (To be used here and on the retake button, would be a good exercise too).